src/main/java/com/rackspacecloud/metrics/queryservice/configuration/CombinedSecurityConfig.java

```diff
@@ -53,11 +58,25 @@ public CombinedSecurityConfig(SecurityProperties securityProperties) {
         protected void configure(HttpSecurity http) throws Exception {
             log.debug("Locking grafana to {}", securityProperties.getWhitelistedIpRange());
             http
-                    .httpBasic().disable()
-                    .csrf().disable()
-                    .antMatcher("/query/**")
-                    .authorizeRequests().antMatchers("/query/**")
+                .httpBasic().disable()
+                .csrf().disable();
+
+            // narrow security filter chain scope for grafana access URLs
+            final HttpSecurity scoped = http.antMatcher("/query/**");
+
+            if (env.acceptsProfiles(Profiles.of("production"))) {
```

Comment:
Is it not better to exclude it for `dev` vs. include it for `production`?

Reply:
...and do we really need a whitelist of ips?  Can it be role based instead?

(not saying it has to be changed in this pr)

Reply:
Good question about exclude for dev vs include for production...I guess since there's three profiles in use in Ceres I went for including the one where it mattered. Otherwise it would have to be "not (development or testing)".

As for whitelisting vs roles, the grafana endpoint is always bypassing repose, even in production. So there isn't a role to authorize but rather authorizing that the request originated from the grafana pod...where the grafana pod is authenticated by SAML (I assume). This probably means the ceres repose config isn't totally ready since it should exclude access to any path isn't the `/v1.0/tenant` path, such as the grafana `/query` path.